src/main/java/com/revature/security/extra/JwtAuthenticationResponse.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.revature.revatureSecurity;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This file "JwtAuthenticationResponse.java" should be located in "com/revature/revatureSecurity" directory, not in "/home/jenkins/workspace/erview-evaluations_PR_PR-57-THYF2QUBWGI6EMGO73GWDKCQJ5HMZBBLF35CQMQWKF6SM2G4YTPA/src/main/java/com/revature/security/extra". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1598)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS00120)